elastio-lambda-stream-backup/app/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# syntax=docker/dockerfile:1
+ARG REGISTRY=public.ecr.aws/elastio-dev/
+
+# Possible values: amd64 or arm64
+ARG ARCH=arm64
+# Version elastio cli what will use like base image
+ARG VERSION_TAG=0.8.0-master
+
+# elastio cli version what be use in the imege(should be the same with red-stack version)
+# to check stack version use command "aws ssm get-parameter --name /elastio/region-stack/version"
```

Comment:
You shouldn't assume that the red stack version and CLI version are always the same.  So far they are but there's no rule that we can't make a CLI release but no red stack release, or vice versa.

Reply:
Okay, i understood what you mean, i will remove this comment.